### ISSUE TITLE
How to evaluate the test results? 

### ISSUE THREAD

**Issue by xinglianglei** [MIC-DKFZ/nnUNet#1868]
Hello, I have a question, that is, I have a dataset(AMOS2022), which includes a training set, a verification set, and a test set. The training set and the verification set are labeled. After I performed a five-fold training crossover using the training set, I got five models. After that, I determined the best configuration. Now I want to test the performance of this configuration on the validation set, but now I can only directly infer the corresponding predicted value of the validation set. How can I calculate the DICE value between the predicted value and the groundtruth? Please forgive my poor English, have a good day.

**Comment by xinglianglei** [comment]
Hi @seziegler 

Can you help me figure it out? Thank you so much

**Comment by seziegler** [comment]
Hi @xinglianglei ,

as described [here](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md#run-inference), commands needed for inference with your determined configuration are printed. You can simply use this command and pass the folder with you validation cases instead of your test cases.

Best,
Sebastian
